resources/wikia/modules/tracker.js

```diff
@@ -108,14 +108,12 @@
 		 *
 		 * @param string event Name of event
 		 * @param object data Extra parameters to track
-		 * @param object successCallback callback function on success (optional)
-		 * @param object errorCallback callback function on failure (optional)
+		 * @param function onComplete callback function when tracking is completed (or fails)
 		 * @param integer timeout How long to wait before declaring the tracking request as failed (optional)
 		 */
-		function internalTrack( event, data, successCallback /* unused */, errorCallback /* unused */, timeout ) {
+		function internalTrack( event, data, onComplete, timeout ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Function has a complexity of 22 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AFunctionComplexity)